src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -360,16 +360,31 @@ Version GetLintVersion (string tool)
 			});
 			var versionInfo = sb.ToString ();
 			if (result != 0 || versionInfo.Contains ("unknown")) {
-				Log.LogWarning ($"Could not get version from '{tool}'");
-				return new Version ();
+				// lets try to parse the lint-xx-x-x-dev.jar filename to get the version
+				var libPath = Path.Combine (Path.GetDirectoryName (tool), "..", "lib");
+				if (Directory.Exists (libPath)) {
+					Version v;
+					foreach (var file in Directory.EnumerateFiles (libPath, "lint-??.?.?-dev.jar")) {
+						var split = Path.GetFileName (file).Split ('-');
+						if (split.Length != 3)
+							continue;
+						if (!string.IsNullOrEmpty (split [1])) {
+							if (Version.TryParse (split [1], out v)) {
+								return v;
+							}
+						}
+					}
+				}
+				Log.LogCodedWarning ("XA0000", $"Could not get version from '{tool}'. Defaulting to 1.0");
```

Comment:
Please select/concoct a new warning code. :-)

Reply:
hence the `do not merge` ;)